basic_pipeline/4_Caps.md

```diff
@@ -0,0 +1,116 @@
+# Introduction
```

Comment:
Let's mark somehow that chapters 4 and 5 can be skipped if someone is not ready for a deep dive into the technology yet.

Reply:
Marking it might be a little bit confusing - formats are needed anyhow in the later chapters (they are used in the caps while defining the further elements). And the text of the 5th chapter (the one about the formats) references the 4th chapter (the one about the caps).